tests/unittests/RecommendationSystemTest.cpp

```diff
@@ -548,6 +554,8 @@ class RecommendationSystemTest : public BackendTest {
     auto internalTypeF = mod.uniqueType(ElemKind::FloatTy, {1});
 
     for (unsigned int i = 0; i < lengths.size(); i++) {
+      Storage *data;
```

Comment:
nit: would prefer to keep this in a more limited scope, i.e. right before `if (enableStaticPlaceholder)` in both cases (so the line will be duplicated). Makes it more clear that it's only used in those scopes.